docs/1_usage.md

```diff
@@ -2,69 +2,53 @@
 
-   ```bash
-   adb shell exit
-   ```
-
-   If you plan to use the root variant, check if you have root access
-
-   ```bash
-   adb shell su -c exit
-   ```
-
-2. Copy the ADB device name
-
-   ```bash
-   adb devices
-   ```
+Put all of that in the same directory for easier usage.
 
 ## 🔨 ReVanced CLI Usage
 
+You may need to slightly modify the commands to adapt to your specific versions of Java and/or Revanced.
+
 - ### Show all available options for the ReVanced CLI
 
   ```bash
-  java -jar revanced-cli.jar -h
+  .\openjdk-19.0.2_windows-x64_bin\jdk-19.0.2\bin\java.exe -jar .\revanced-cli-2.20.0-all.jar -h
   ```
 
 - ### List all available patches from supplied patch bundles
 
   ```bash
-  java -jar revanced-cli.jar \
-      -b revanced-patches.jar \
-      -l
+  .\openjdk-19.0.2_windows-x64_bin\jdk-19.0.2\bin\java.exe -jar .\revanced-cli-2.20.0-all.jar -b .\revanced-patches-2.161.1.jar -l --apk '.\com.google.android.youtube_18.03.36-1535632832_minAPI26(arm64-v8a,armeabi-v7a,x86,x86_64)(nodpi)_apkmirror.com.apk' --with-packages --with-versions
   ```
 
-- ### Use the ReVanced CLI without root permissions
+- ### Generate the APK with the patches you want
 
-  ```bash
-  java -jar revanced-cli.jar \
-   -a input.apk \
-   -c \
-   -o unpatched-output.apk \
-   -b revanced-patches.jar
-  ```
+```bash
+<java> -jar <revanced-cli.jar> -b <revanced-patches.jar> --apk <com.example.apk> -m <revanced-integrations.apk> --exclusive --out <output.apk> -i <patch1> [-i patchX]
+```
 
-- ### Mount the patched application with root permissions over the installed application
+For example:
 
   ```bash
-  adb install input.apk # make sure the same version is installed
-  java -jar revanced-cli.jar \
-      -a input.apk \
-      -c \
-      -d device-name \
-      -o patched-output.apk \
-      -b revanced-patches.jar \
-      -e microg-support \
-      --mount
+  .\openjdk-19.0.2_windows-x64_bin\jdk-19.0.2\bin\java.exe -jar .\revanced-cli-2.20.0-all.jar -b .\revanced-patches-2.161.1.jar --apk '.\com.google.android.youtube_18.03.36-1535632832_minAPI26(arm64-v8a,armeabi-v7a,x86,x86_64)(nodpi)_apkmirror.com.apk' -m .\revanced-integrations-0.96.1.apk --exclusive --out yt-revanced.apk -i video-ads -i general-ads -i hide-my-mix -i client-spoof -i sponsorblock -i hide-watermark -i custom-branding -i hide-info-cards -i seekbar-tapping -i hide-album-cards -i hide-artist-card -i hide-cast-button -i hide-watch-in-vr -i spoof-app-version -i custom-video-speed -i hide-shorts-button -i minimized-playback -i old-quality-layout -i open-links-directly -i disable-zoom-haptics -i hide-endscreen-cards -i hide-crowdfunding-box -i return-youtube-dislike -i hide-breaking-news-shelf -i disable-auto-player-popup-panels -i disable-fullscreen-panels -i disable-startup-shorts-player -i remove-player-button-background -i enable-wide-searchbar -i microg-support
```

Comment:
Now the guide is constrained to YouTube and includes all current patches, whereas the CLI includes the recommended ones by default. Same issues as comments above.